### PR TITLE
pytest_embedded_arduino/: Separated flash settings for each target to fix H2

### PR DESCRIPTION
**Pull request by P-R-O-C-H-Y**
I have separated flash_setting for each SoC, as the H2 cannot flash on default 80MHz.
Tested locally on H2.

### CODE REVIEW

File: pytest-embedded-arduino/pytest_embedded_arduino/app.py [hunk 1]
```diff
@@ -17,7 +17,14 @@ class ArduinoApp(App):
     """
 
     #: dict of flash settings
-    flash_settings = {'flash_mode': 'dio', 'flash_size': 'detect', 'flash_freq': '80m'}
+    flash_settings = {
+    	'esp32': {'flash_mode': 'dio', 'flash_size': 'detect', 'flash_freq': '80m'},
```
Comment (hfudev) [inline pytest-embedded-arduino/pytest_embedded_arduino/app.py:21]: here this line is mixed with spaces and tabs

**Review by hfudev — CHANGES_REQUESTED** [review]
Overall LGTM! the pre-commit check is failing. Could you run `pre-commit install` locally and run the hook? that should help fix the mixed spaces and tabs issue.

Besides, could you follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)? The commit message is used for generating the changelog.

For example, the commit message could be `fix(arduino): separated flash settings for each target`

File: pytest-embedded-arduino/pytest_embedded_arduino/app.py (hunk 1 above)
Comment (P-R-O-C-H-Y) [reply pytest-embedded-arduino/pytest_embedded_arduino/app.py]: Should be fixed by running pre-commit. As a commit message I have pasted what you suggested co you can squash both commits and keep the last RIGHT message. Sorry for that.

**Comment by P-R-O-C-H-Y** [comment]
> @P-R-O-C-H-Y Mind squash the commits? Thanks! The reason is that the github feature "squash and merge" will make the commit message a bit dirty. [ref](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#merge-message-for-a-squash-merge)
> 
> The commit message would be
> 
> ```
> the pull request title + number
> 
> - commit 1 message
> - commit 2 message
> ```

Sorry about that. I need to be more careful.. Should Iclose this PR and open new one with the right commit messages?

**Comment by P-R-O-C-H-Y** [comment]
@hfudev Yeah I meant the squash and merge, but I did not know that it will make the message dirty. I have squashed the commits and force pushed. Should be fine now right? :)

**Comment by hfudev** [comment]
@P-R-O-C-H-Y Thanks!

**Pull request merged**